apps/wallet-mobile/src/features/Notifications/useCases/common/NotificationPopup.tsx

```diff
@@ -108,7 +108,9 @@ const useStyles = () => {
       ...atoms.p_lg,
       ...atoms.gap_lg,
       ...atoms.flex_row,
-      backgroundColor: color.bg_color_max,
+      ...atoms.border,
+      backgroundColor: isLight ? color.bg_color_max : color.gray_100,
+      borderColor: color.gray_50,
```

Comment:
I think we're missing a shadow as well? We don't have them as colors in the palette though.